content/docs/forms.md

```diff
@@ -182,13 +182,49 @@ class FlavorForm extends React.Component {
 
 Overall, this makes it so that `<input type="text">`, `<textarea>`, and `<select>` all work very similarly - they all accept a `value` attribute that you can use to implement a controlled component.
 
-> Note
->
-> You can pass an array into the `value` attribute, allowing you to select multiple options in a `select` tag:
->
->```js
-><select multiple={true} value={['B', 'C']}>
->```
+### The select multiple Tag
+Just like in HTML, select elements can become multi-select elements through the addition of the *multiple* attribute to the select tag. But handling of the value attribute in react is more complex due to having an array of values. 
+
+Here's our example select element with multi-select support:
+
+```js{4,11,24}
+class FlavorForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {value: ['coconut']};
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({value: Array.from(event.target.options).filter(o => o.selected).map(o => o.value)});
+  }
+
+  handleSubmit(event) {
+    alert('Your favorite flavor is: ' + this.state.value);
```

Comment:
It might make a bit more sense if this used a more multiple + plural example, like "Your favorite flavors are" or "You picked: ..."

Reply:
I thought about that, but then I thought if this is my favorite flavor of ice cream then it would be multiple flavors. So the language actually makes sense in the current context.